packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -132,12 +132,9 @@ class CreateCommand extends FlutterCommand {
       defaultsTo: 'kotlin',
       allowed: <String>['java', 'kotlin'],
     );
-    // TODO(egarciad): Remove this flag. https://github.com/flutter/flutter/issues/52363
     argParser.addFlag(
-      'androidx',
+      'skip-checks',
```

Comment:
Why did we need this flag? Is there a scenario where we want to not validate project names?

Reply:
Just for our own integration tests

Reply:
Technically we can't stop people from changing the name after the fact, and our tooling needs to work with arbitrary names

Reply:
I see. Maybe `skip-name-checks`? And add a "help" message to explain such?

Reply:
done, though it is still hidden since it is only for internal use